### PR TITLE
feat(laravel): add alias for `optimize:clear`

### PR DESCRIPTION
**Pull request by texe**
## Standards checklist:

- [x] The PR title is descriptive.
- [ ] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- Added alias paopc for the php artisan optimize:clear command in the Laravel plugin.

## Other comments:
This alias provides a quick way to clear all cached data, streamlining the process and ensuring that the application runs with the latest configuration and optimized settings.

...

### CODE REVIEW

**Review by carlosala — COMMENTED** [review]
Hi! Looks good to me! Could you update the README accordingly? Thanks!

**Comment by texe** [comment]
> Hi! Looks good to me! Could you update the README accordingly? Thanks!

Hi! I've updated the README to include the paopc alias as requested. Let me know if there's anything else to adjust. Thanks! 😊

File: plugins/laravel/laravel.plugin.zsh [hunk 1]
```diff
@@ -36,6 +36,8 @@ alias pacac='php artisan cache:clear'
 alias pacoc='php artisan config:clear'
 alias pavic='php artisan view:clear'
 alias paroc='php artisan route:clear'
+alias paopc='php artisan optimize:clear'
+
```
Comment (carlosala) [inline plugins/laravel/laravel.plugin.zsh:40]:
```suggestion
```

**Review by carlosala — APPROVED** [review]

**Comment by carlosala** [comment]
Thanks for your contribution!

**Pull request merged**